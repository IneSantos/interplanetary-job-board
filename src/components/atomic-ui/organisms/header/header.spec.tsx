import React from "react";
import { render } from '@testing-library/react';
import Header from "./header";

describe("Header atom tests", () => {
  test(`Header component render`, () => {
    const { container } = render(
      <Header searchInput="This is an input" handleSearch={jest.fn()}/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
