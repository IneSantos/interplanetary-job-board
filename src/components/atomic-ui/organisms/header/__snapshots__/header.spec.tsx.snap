// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header atom tests Header component render 1`] = `
<div
  class="header-wrapper"
>
  <div
    class="search"
  >
    <button
      class="search-button"
      type="submit"
    >
      <i
        class="fas fa-search"
      />
    </button>
    <input
      class="search-term"
      placeholder="What are you looking for?"
      type="text"
      value="This is an input"
    />
  </div>
</div>
`;
